openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeOperations.kt

```diff
@@ -572,8 +652,14 @@ fun intersections(a: Shape, b: Shape) : List<ContourIntersection> {
 }
 
 
-
-
+/**
+ * Splits a [Shape] into two separate [Shape]s from given [LineSegment].
+ *
+ * [LineSegment] doesn't necessarily need to cover the full length of the [Shape],
+ * as it will be extended on both ends to ensure it splits the whole [Shape].
+ *
+ * @return A pair containing two partial [Shape]s.
+ */
 fun split(shape: Shape, line: LineSegment): Pair<Shape, Shape> {
```

Comment:
Maybe `line` should be `cutter` as in the next functions?

Reply:
Out of scope for this PR :(